### PR TITLE
fix(apps/hermes/client): allow baseURL with suffixes

### PR DESCRIPTION
**Pull request by mubaris**

### CODE REVIEW

File: apps/hermes/client/js/src/HermesClient.ts [hunk 1]
```diff
@@ -100,7 +100,7 @@ export class HermesClient {
     query?: string;
     filter?: string;
   }): Promise<PriceFeedMetadata[]> {
-    const url = new URL("/v2/price_feeds", this.baseURL);
+    const url = new URL(`${this.baseURL}/v2/price_feeds`);
```
Comment (cctdaniel) [inline apps/hermes/client/js/src/HermesClient.ts:103]: A more robust approach would be to ensure that `this.baseURL` always ends with a slash before appending the path:

e.g.
```
const baseURL = this.baseURL.endsWith('/') ? this.baseURL : `${this.baseURL}/`;
const url = new URL('v2/price_feeds', baseURL);
```

The URL constructor ensures that the protocol and host parts of the base URL are preserved correctly, even if the base URL has additional path components.

If you could also make the changes for `getLatestPriceUpdates` and `getPriceUpdatesAtTimestamp` that would be appreciated!

**Review by cctdaniel — CHANGES_REQUESTED** [review]
Thanks for catching the bug and opening a PR, left a comment and happy to look again once that's addressed.

File: apps/hermes/client/js/src/HermesClient.ts (hunk 1 above)
Comment (mubaris) [reply apps/hermes/client/js/src/HermesClient.ts]: Have updated it

**Review by cctdaniel — APPROVED** [review]

**Comment by cctdaniel** [comment]
thanks for your contribution! can you bump the package version?

**Comment by mubaris** [comment]
done!

**Pull request merged**